script/nats-controller.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
you should be able to reuse the `binary-controller` script instead of adding a new one